Assignment 3.Rmd

```diff
@@ -26,20 +31,20 @@ The algorithm will treat each row as a value belonging to a person, so we need t
 
 ```{r}
 
-K2 <- 
+K2 <- K1[,-1]
 
 ```
 
-It is important to think about the meaning of missing values when clustering. We could treat them as having meaning or we could remove those people who have them. Neither option is ideal. What problems do you foresee if we recode or remove these values? Write your answers below:
-
+Q1: It is important to think about the meaning of missing values when clustering. We could treat them as having meaning or we could remove those people who have them. Neither option is ideal. What problems do you foresee if we recode or remove these values? Write your answers below:
 
+A1: If there are a lot of obersvations with missing values relative to total observations, removing them would make the cluster analysis less accurate. K-means clustering is sensitive to oultiers so this is especially true if removal creates artificial ones. Recoding missing values to expected ones reinforces the existing cluster pattern and removes the possibility of uncovering a more accurate one, ultimately compromising the validity of the model. 
```

Comment:
A good answer, but you need to be more specific. What do you mean by "less accurate" and "reinforces the existing cluster pattern"?